app/views/works/show.html.erb

```diff
@@ -174,17 +174,17 @@
     </p>
   <% end %>
 
-  <% if @work.resource.funder_name.present? %>
-    <p>Funder Name: <%= @work.resource.funder_name %></p>
-  <% end %>
-
-  <% if @work.resource.award_number.present? %>
-    <p>Award Number: <%= @work.resource.award_number %></p>
-  <% end %>
-
-  <% if @work.resource.award_uri.present? %>
-    <p>Award URI: <%= @work.resource.award_uri %></p>
-  <% end %>
+  <% if @work.resource.funders.count > 0 %>
+    <p>Funders:
+      <% @work.resource.funders.each do |funder| %>
```

Comment:
Are we loosing the Headings here? `Funder Name` `Award Number` `Award URI:`

Reply:
I had thought it might be clear enough from context: Each subfield has a distinct format. Since this will repeat, I'm not sure if this would be better presented as a table. 
- Filed #785 for follow-up.